lib/fog/linode/models/compute/kernels.rb

```diff
@@ -11,20 +11,27 @@ def all
           load kernels
         end
 
+        def all_kvm
+          load kernels(:isKVM => 1)
+        end
+
+        def all_xen
+          load kernels(:isXen => 1)
+        end
```

Comment:
@plribeiro3000 using `options` on a single `all` method would also be better if you wanted to dynamically select either KVM or Xen kernels. I have committed e73242e32a68008bcecaa669df23cb6af266ccb8 to remove `all_kvm` and `all_xen` and add an `options` parameter to `all`.